test/fixtures/wpt/url/resources/urltestdata.json

```diff
@@ -5389,84 +5389,84 @@
   {
     "input": "file:\\\\//",
     "base": "about:blank",
-    "href": "file:///",
+    "href": "file:////",
     "protocol": "file:",
     "username": "",
     "password": "",
     "host": "",
     "hostname": "",
     "port": "",
-    "pathname": "/",
+    "pathname": "//",
     "search": "",
     "hash": ""
   },
   {
     "input": "file:\\\\\\\\",
     "base": "about:blank",
-    "href": "file:///",
+    "href": "file:////",
     "protocol": "file:",
     "username": "",
     "password": "",
     "host": "",
     "hostname": "",
     "port": "",
-    "pathname": "/",
+    "pathname": "//",
     "search": "",
     "hash": ""
   },
   {
     "input": "file:\\\\\\\\?fox",
     "base": "about:blank",
-    "href": "file:///?fox",
+    "href": "file:////?fox",
     "protocol": "file:",
     "username": "",
     "password": "",
     "host": "",
     "hostname": "",
     "port": "",
-    "pathname": "/",
+    "pathname": "//",
     "search": "?fox",
     "hash": ""
   },
   {
     "input": "file:\\\\\\\\#guppy",
     "base": "about:blank",
-    "href": "file:///#guppy",
+    "href": "file:////#guppy",
     "protocol": "file:",
     "username": "",
     "password": "",
     "host": "",
     "hostname": "",
     "port": "",
-    "pathname": "/",
+    "pathname": "//",
     "search": "",
     "hash": "#guppy"
   },
   {
     "input": "file://spider///",
     "base": "about:blank",
-    "href": "file://spider/",
+    "href": "file://spider///",
     "protocol": "file:",
     "username": "",
     "password": "",
     "host": "spider",
     "hostname": "spider",
     "port": "",
-    "pathname": "/",
+    "pathname": "///",
```

Comment:
This is a little confusing to me. Why are the `/` in paths no longer being deduped? I am concerned this will be breaking for modules users who make this mistake.